app/models/user.rb

```diff
@@ -30,6 +30,10 @@ def clubhouses_as_member
     clubhouses_of_role(Membership::MEMBER_ROLE)
   end
 
+  def member_of?(clubhouse)
+    !memberships.find_by(clubhouse:).nil?
+  end
```

Comment:
Rails provides the `#present?` and `#blank?` methods. They are more intuitive than ruby's `#nil?` method
https://blog.appsignal.com/2018/09/11/differences-between-nil-empty-blank-and-present.html

```suggestion
  def member_of?(clubhouse)
    memberships.find_by(clubhouse:).present?
  end
```